src/styles/site.css

```diff
@@ -272,6 +272,10 @@ img, svg {
         display: inline-block;
     }
 
+    .img-home {
```

Comment:
I'm familiar with the way Tailwind is used in this codebase so happy to hear what the convention to naming this class should be.

Reply:
For now there is no convention at all, so this is totally fine :-)